migration/migration.go

```diff
@@ -472,9 +475,10 @@ func insertDirectoriesAsIdpConfigs(ctx context.Context, mongodb *MongoDB, postgr
 	dbName := "platform-db"
 	collectionName := "Directory"
 	reqLogger := logf.FromContext(ctx).WithValues("MongoDB.DB", dbName, "MongoDB.Collection", collectionName)
+	migrationKey := postgres.GetMigrationKey()
 	filter := bson.M{
 		"$and": bson.A{
-			bson.M{"migrated": bson.M{"$ne": true}},
+			bson.M{migrationKey: bson.M{"$ne": true}},
```

Comment:
where we have IDP and scimserver it will have only migrationKey?

Reply:
We are not writing back the migration flag to idp and scimserver